scripts/totalopenstation-cli-parser.py

```diff
@@ -87,10 +89,33 @@
     dest="list",
     default=False,
     help=_("list the available input and output formats"))
+parser.add_option(
+                "--log",
+                action="store",
+                dest="loglevel",
+                default="WARNING",
+                type="string",
+                help=_("minimum log level"))
+parser.add_option(
+                "--logtofile",
+                action="store_true",
+                dest="logotfile",
+                default=False,
+                help=_("log to file"))
 
 
 (options, args) = parser.parse_args()
 
+logger = logging.getLogger("tops")
```

Comment:
The logger should absolutely be called "totalopenstation"! :grin:

Reply:
Hahaha will change it.

Reply:
Done